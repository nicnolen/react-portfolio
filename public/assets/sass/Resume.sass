@import './variables'
@import './index'
@import './App'

.resume 
  font-size: 5vh
  text-decoration: none

.wrapper h4
  color: $red
  font-size: 2.5vh
  
li, li a, div a
  color: $light-blue

ul, .wrapper li a, .download-resume
  font-size: 2vh 