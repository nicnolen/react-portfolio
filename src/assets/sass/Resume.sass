@import './variables'
@import './index'
@import './App'

.resume 
  font-size: 5vh
  text-decoration: none

.skills h3, h4 
  color: $red
  