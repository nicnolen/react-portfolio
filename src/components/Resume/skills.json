[
  {
    "name": "Git Bash",
    "category": "frontend",
    "documentation": "https://git-scm.com/docs"
  },
  {
    "name": "GitHub",
    "category": "frontend",
    "documentation": "https://docs.github.com/en/get-started/quickstart/set-up-git"
  },
  {
    "name": "Markdown",
    "category": "frontend",
    "documentation": "https://www.markdownguide.org/basic-syntax"
  },
  {
    "name": "HTML",
    "category": "frontend",
    "documentation": "https://www.w3schools.com/html/"
  },
  {
    "name": "CSS",
    "category": "frontend",
    "documentation": "https://www.w3schools.com/css/"
  },
  {
    "name": "Sass",
    "category": "frontend",
    "documentation": "https://sass-lang.com/documentation"
  },
  {
    "name": "Bootstrap",
    "category": "frontend",
    "documentation": "https://getbootstrap.com/docs/4.1/getting-started/introduction/"
  },
  {
    "name": "Bulma",
    "category": "frontend",
    "documentation": "https://bulma.io/documentation/"
  },
  {
    "name": "Iconic",
    "category": "frontend",
    "documentation": "https://iconicwp.com/docs/"
  },
  {
    "name": "Font Awesome",
    "category": "frontend",
    "documentation": "https://fontawesome.com/docs/web/setup/get-started"
  },
  {
    "name": "JavaScript",
    "category": "frontend",
    "documentation": "https://www.w3schools.com/js/default.asp"
  },
  {
    "name": "JSON",
    "category": "frontend",
    "documentation": "https://www.json.org/json-en.html"
  },
  {
    "name": "React.js",
    "category": "frontend",
    "documentation": "https://reactjs.org/docs/getting-started.html"
  },
  {
    "name": "Webpack",
    "category": "frontend",
    "documentation": "https://webpack.js.org/concepts/"
  },
  {
    "name": "Handlebars.js",
    "category": "frontend",
    "documentation": "https://handlebarsjs.com/guide/"
  },
  {
    "name": "jQuery",
    "category": "frontend",
    "documentation": "https://api.jquery.com/"
  },
  {
    "name": "Moment.js",
    "category": "frontend",
    "documentation": "https://momentjs.com/"
  },
  {
    "name": "Shell Script",
    "category": "frontend",
    "documentation": "https://devdocs.io/bash/"
  },
  {
    "name": "Adobe Photoshop",
    "category": "frontend",
    "documentation": "https://helpx.adobe.com/photoshop/user-guide.html"
  },
  {
    "name": "SPAs",
    "category": "frontend",
    "documentation": "https://developer.mozilla.org/en-US/docs/Glossary/SPA"
  },
  {
    "name": "Heroku",
    "category": "backend",
    "documentation": "https://devcenter.heroku.com/categories/reference"
  },
  {
    "name": "Node.js",
    "category": "backend",
    "documentation": "https://nodejs.org/en/docs/"
  },
  {
    "name": "Express.js",
    "category": "backend",
    "documentation": "https://expressjs.com/en/5x/api.html"
  },
  {
    "name": "MySQL",
    "category": "backend",
    "documentation": "https://dev.mysql.com/doc/refman/8.0/en/dynindex-statement.html"
  },
  {
    "name": "Sequelize",
    "category": "backend",
    "documentation": "https://sequelize.org/v7/manual/getting-started.html"
  },
  {
    "name": "JawsDB",
    "category": "backend",
    "documentation": "https://www.jawsdb.com/docs/#mysql"
  },
  {
    "name": "MongoDB",
    "category": "backend",
    "documentation": "https://www.mongodb.com/docs/manual/tutorial/getting-started/"
  },
  {
    "name": "Mongoose",
    "category": "backend",
    "documentation": "https://mongoosejs.com/docs/api.html"
  },
  {
    "name": "MongoDB Atlas",
    "category": "backend",
    "documentation": "https://www.mongodb.com/docs/atlas/getting-started/"
  },
  {
    "name": "Inquirer",
    "category": "backend",
    "documentation": "https://www.npmjs.com/package/inquirer#documentation"
  },
  {
    "name": "sessionstorage",
    "category": "backend",
    "documentation": "https://www.npmjs.com/package/sessionstorage"
  },
  {
    "name": "bcrypt",
    "category": "backend",
    "documentation": "https://www.npmjs.com/package/bcrypt"
  },
  {
    "name": "dotenv",
    "category": "backend",
    "documentation": "https://www.npmjs.com/package/dotenv"
  },
  {
    "name": "Jest",
    "category": "backend",
    "documentation": "https://jestjs.io/docs/getting-started"
  },
  {
    "name": "Insomnia",
    "category": "backend",
    "documentation": "https://docs.insomnia.rest/"
  },
  {
    "name": "Benchmark.js",
    "category": "backend",
    "documentation": "https://benchmarkjs.com/docs"
  },
  {
    "name": "PWAs",
    "category": "backend",
    "documentation": "https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps"
  },
  {
    "name": "TDD",
    "category": "backend",
    "documentation": "https://en.wikipedia.org/wiki/Test-driven_development"
  },
  {
    "name": "OOP",
    "category": "backend",
    "documentation": "https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object-oriented_programming"
  },
  {
    "name": "MVC",
    "category": "backend",
    "documentation": "https://developer.mozilla.org/en-US/docs/Glossary/MVC"
  },
  {
    "name": "REST APIs",
    "category": "backend",
    "documentation": "https://developer.mozilla.org/en-US/docs/Glossary/REST"
  },
  {
    "name": "ORM",
    "category": "backend",
    "documentation": "https://en.wikipedia.org/wiki/Object%E2%80%93relational_mapping"
  },
  {
    "name": "ODM",
    "category": "backend",
    "documentation": "https://www.tutorialspoint.com/phalcon/phalcon_object_document_mapper.htm"
  },
  {
    "name": "Communication",
    "category": "technical"
  },
  {
    "name": "Time Management",
    "category": "technical"
  },
  {
    "name": "Problem Solving",
    "category": "technical"
  },
  {
    "name": "Group Work",
    "category": "technical"
  },
  {
    "name": "Self-Starter",
    "category": "technical"
  }
]
