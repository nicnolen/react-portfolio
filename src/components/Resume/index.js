import React from 'react';
import '../../assets/css/Resume.css';
import resume from '../../assets/images/portfolioImages/resume.pdf';
import FrontendSkills from '../FrontendSkills';
import BackendSkills from '../BackendSkills';
import TechnicalSkills from '../TechnicalSkills';
import skillsInfo from './skills.json';

function Resume() {
  const Wrapper = props => {
    return <div className='wrapper'>{props.children}</div>;
  };
  return (
    <section className='container'>
      <h2 className='top-title'>Resume</h2>
      <div>
        <a href={resume} target='_blank' rel='noreferrer'>
          Download Resume
        </a>
      </div>
      <div>
        <h3>Skills</h3>
        <Wrapper id='card-data'>
          <h4>Front End Skills</h4>
          {skillsInfo.map((skill, i) => (
            <FrontendSkills
              key={i}
              name={skill.name}
              category={skill.category}
              documentation={skill.documentation}
            />
          ))}
          <h4>Back End Skills</h4>
          {skillsInfo.map((skill, i) => (
            <BackendSkills
              key={i}
              name={skill.name}
              category={skill.category}
              documentation={skill.documentation}
            />
          ))}
          <h4>Technical Skills</h4>
          {skillsInfo.map((skill, i) => (
            <TechnicalSkills
              key={i}
              name={skill.name}
              category={skill.category}
              documentation={skill.documentation}
            />
          ))}
        </Wrapper>
      </div>
    </section>
  );
}

export default Resume;
