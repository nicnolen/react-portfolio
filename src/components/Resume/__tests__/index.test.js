// Import dependencies
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Resume from '..';

// After each test, make sure there isnt any leftover memory that could give you false results
afterEach(cleanup);

// Declare the components you are testing
describe('Resume component', () => {
  // verify the component is rendering properly
  it('Resume component renders', () => {
    render(<Resume />);
  });

  // compare snapshot versions of the DOM node structure
  it('matches snapshot DOM node structure', () => {
    const { asFragment } = render(<Resume />);
    expect(asFragment()).toMatchSnapshot();
  });
});