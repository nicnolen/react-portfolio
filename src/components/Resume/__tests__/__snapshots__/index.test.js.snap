// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section
    class="container pb-5"
  >
    <h2
      class="top-title"
    >
      Resume
    </h2>
    <div
      class="download-resume"
    >
      <a
        href="resume.pdf"
        rel="noreferrer"
        target="_blank"
      >
        <h3
          class="pb-5"
        >
          Download Resume
        </h3>
      </a>
    </div>
    <div>
      <h3
        class="subtitle text-center pb-4"
      >
        Skills
      </h3>
      <div
        class="wrapper"
      >
        <h4
          class="pb-3"
        >
          Front End Skills (Click on a skill for more information)
        </h4>
        <div>
          <ul>
            <li>
              <a
                href="https://git-scm.com/docs"
                rel="noreferrer"
                target="_blank"
              >
                Git Bash
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://docs.github.com/en/get-started/quickstart/set-up-git"
                rel="noreferrer"
                target="_blank"
              >
                GitHub
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.markdownguide.org/basic-syntax"
                rel="noreferrer"
                target="_blank"
              >
                Markdown
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.w3schools.com/html/"
                rel="noreferrer"
                target="_blank"
              >
                HTML
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.w3schools.com/css/"
                rel="noreferrer"
                target="_blank"
              >
                CSS
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://sass-lang.com/documentation"
                rel="noreferrer"
                target="_blank"
              >
                Sass
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://getbootstrap.com/docs/4.1/getting-started/introduction/"
                rel="noreferrer"
                target="_blank"
              >
                Bootstrap
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://bulma.io/documentation/"
                rel="noreferrer"
                target="_blank"
              >
                Bulma
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://iconicwp.com/docs/"
                rel="noreferrer"
                target="_blank"
              >
                Iconic
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://fontawesome.com/docs/web/setup/get-started"
                rel="noreferrer"
                target="_blank"
              >
                Font Awesome
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.w3schools.com/js/default.asp"
                rel="noreferrer"
                target="_blank"
              >
                JavaScript
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.json.org/json-en.html"
                rel="noreferrer"
                target="_blank"
              >
                JSON
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://reactjs.org/docs/getting-started.html"
                rel="noreferrer"
                target="_blank"
              >
                React.js
              </a>
            </li>
          </ul>
        </div>
        <div />
        <div>
          <ul>
            <li>
              <a
                href="https://webpack.js.org/concepts/"
                rel="noreferrer"
                target="_blank"
              >
                Webpack
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://handlebarsjs.com/guide/"
                rel="noreferrer"
                target="_blank"
              >
                Handlebars.js
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://api.jquery.com/"
                rel="noreferrer"
                target="_blank"
              >
                jQuery
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://momentjs.com/"
                rel="noreferrer"
                target="_blank"
              >
                Moment.js
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://devdocs.io/bash/"
                rel="noreferrer"
                target="_blank"
              >
                Shell Script
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://eslint.org/docs/user-guide/configuring/"
                rel="noreferrer"
                target="_blank"
              >
                ESLint
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://prettier.io/docs/en/options.html"
                rel="noreferrer"
                target="_blank"
              >
                Prettier
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://cloudinary.com/documentation"
                rel="noreferrer"
                target="_blank"
              >
                Cloudinary
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://helpx.adobe.com/photoshop/user-guide.html"
                rel="noreferrer"
                target="_blank"
              >
                Adobe Photoshop
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://developer.mozilla.org/en-US/docs/Glossary/SPA"
                rel="noreferrer"
                target="_blank"
              >
                SPAs
              </a>
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://dev.mobify.com/v2.x/how-to-guides/categories/state-management/managing-global-state"
                rel="noreferrer"
                target="_blank"
              >
                Global State
              </a>
            </li>
          </ul>
        </div>
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <h4
          class="pb-3"
        >
          Back End Skills (Click on a skill for more information)
        </h4>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://reactrouter.com/docs/en/v6"
                rel="noreferrer"
                target="_blank"
              >
                React Router
              </a>
            </li>
          </ul>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://docs.github.com/en/actions/quickstart"
                rel="noreferrer"
                target="_blank"
              >
                GitHub Actions
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://yaml.org/spec/1.2.2/"
                rel="noreferrer"
                target="_blank"
              >
                YAML
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://devcenter.heroku.com/categories/reference"
                rel="noreferrer"
                target="_blank"
              >
                Heroku
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://nodejs.org/en/docs/"
                rel="noreferrer"
                target="_blank"
              >
                Node.js
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://expressjs.com/en/5x/api.html"
                rel="noreferrer"
                target="_blank"
              >
                Express.js
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://dev.mysql.com/doc/refman/8.0/en/dynindex-statement.html"
                rel="noreferrer"
                target="_blank"
              >
                MySQL
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://sequelize.org/v7/manual/getting-started.html"
                rel="noreferrer"
                target="_blank"
              >
                Sequelize
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.jawsdb.com/docs/#mysql"
                rel="noreferrer"
                target="_blank"
              >
                JawsDB
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.mongodb.com/docs/manual/tutorial/getting-started/"
                rel="noreferrer"
                target="_blank"
              >
                MongoDB
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://mongoosejs.com/docs/api.html"
                rel="noreferrer"
                target="_blank"
              >
                Mongoose
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.mongodb.com/docs/atlas/getting-started/"
                rel="noreferrer"
                target="_blank"
              >
                MongoDB Atlas
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://javascript.info/indexeddb"
                rel="noreferrer"
                target="_blank"
              >
                IndexedDB
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.apollographql.com/docs/"
                rel="noreferrer"
                target="_blank"
              >
                Apollo
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://graphql.org/learn/"
                rel="noreferrer"
                target="_blank"
              >
                GraphQL
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://stripe.com/docs"
                rel="noreferrer"
                target="_blank"
              >
                Stripe
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://fakerjs.dev/guide/"
                rel="noreferrer"
                target="_blank"
              >
                Faker.js
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.npmjs.com/package/inquirer#documentation"
                rel="noreferrer"
                target="_blank"
              >
                Inquirer
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.npmjs.com/package/sessionstorage"
                rel="noreferrer"
                target="_blank"
              >
                sessionstorage
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://jwt.io/introduction"
                rel="noreferrer"
                target="_blank"
              >
                JSON Web Token (JWT)
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.npmjs.com/package/bcrypt"
                rel="noreferrer"
                target="_blank"
              >
                bcrypt
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.npmjs.com/package/dotenv"
                rel="noreferrer"
                target="_blank"
              >
                dotenv
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://jestjs.io/docs/getting-started"
                rel="noreferrer"
                target="_blank"
              >
                Jest
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://docs.insomnia.rest/"
                rel="noreferrer"
                target="_blank"
              >
                Insomnia
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://benchmarkjs.com/docs"
                rel="noreferrer"
                target="_blank"
              >
                Benchmark.js
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps"
                rel="noreferrer"
                target="_blank"
              >
                PWAs
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://en.wikipedia.org/wiki/Test-driven_development"
                rel="noreferrer"
                target="_blank"
              >
                TDD
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object-oriented_programming"
                rel="noreferrer"
                target="_blank"
              >
                OOP
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://developer.mozilla.org/en-US/docs/Glossary/MVC"
                rel="noreferrer"
                target="_blank"
              >
                MVC
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://developer.mozilla.org/en-US/docs/Glossary/REST"
                rel="noreferrer"
                target="_blank"
              >
                REST APIs
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://en.wikipedia.org/wiki/Object%E2%80%93relational_mapping"
                rel="noreferrer"
                target="_blank"
              >
                ORM
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.tutorialspoint.com/phalcon/phalcon_object_document_mapper.htm"
                rel="noreferrer"
                target="_blank"
              >
                ODM
              </a>
            </li>
          </ul>
           
        </div>
        <div>
          <ul>
            <li>
              <a
                href="https://www.tutorialspoint.com/phalcon/phalcon_object_document_mapper.htm"
                rel="noreferrer"
                target="_blank"
              >
                ODM
              </a>
            </li>
          </ul>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <div>
           
        </div>
        <h4
          class="pb-3"
        >
          Technical Skills
        </h4>
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div>
          <ul>
            <li>
              Communication
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              Time Management
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              Problem Solving
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              Group Work
            </li>
          </ul>
        </div>
        <div>
          <ul>
            <li>
              Self-Starter
            </li>
          </ul>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
